pkg/hintrunner/zero/README.md

```diff
@@ -49,7 +49,8 @@ func newAssert250bitsHint(low, high, value hinter.ResOperander) hinter.Hinter {
 5- This is the part that gets insteresting, because implementing the hint means you have to write a code in go that has the same behavior as the code of the hint in python. The difficulty of course depends on the hint. Some things that are good to know here:
 
 - It might be the case that the hint is already implemented as part of the **core** pkg (Cairo 1 hints), so you might want to check first if it is. Such is the case for `AssertLeFelt` hint. In those cases you would only need to return the corresponding hint structure from the **core** pkg.
-- The operanders allow you to get the address in memory of the variable you need to use. When this variable is a felt just reading the value from that address is enough, but sometimes the variable is actually a struct so you need to get several consecutive values in memory, for that you can use `GetConsecutiveMemoryValues(addr MemoryAddress, size int16)` method. For example in `uint256add` hint you need to add `a` and `b` which are both `Uint256` structs, so we used `GetUint256AsFelts` utility which basically calls that function with size 2.
+- The operanders allow you to get the address in memory of the variable you need to use. When this variable is a `felt`, simply reading the value from that address is sufficient. However, if the variable is a `struct`, you need to retrieve several consecutive values from memory. For this, you can use the `GetConsecutiveMemoryValues(addr MemoryAddress, size int16)` method. 
```

Comment:
> For this, you can use the `GetConsecutiveMemoryValues(addr MemoryAddress, size int16)` method.

For this, you can use the `GetConsecutiveMemoryValues(addr MemoryAddress, size int16)` method, which calls `ReadFromAddress` function inside a loop.

I feel like the flow is not good. We talk about `GetConsecutiveMemoryValues` , and then, we say '' for example '' to explain that `uint256add` doesnt use `GetConsecutiveMemoryValues` .

Can you reformulate in some way ?

Reply:
🤔Let me see what I can do